README.md

```diff
@@ -80,6 +80,21 @@ A high level breakdown of the components of the framework:
 * [mgfxc](Tools/MonoGame.Effect.Compiler) is the command line effect compiler tool.
 * The [mgcb-editor](Tools/MonoGame.Content.Builder.Editor) tool is a GUI frontend for content processing.
 
+### Syncing/Submitting PRs
```

Comment:
Thinking about it, we probably shouldn't have this in the README, this does not give detailed enough information for a new person and for someone who knows how to use git is not useful.